python/test/cambi_test.py

```diff
@@ -45,9 +45,9 @@ def test_run_cambi_fextractor_scaled(self):
 
         # score: arithmetic mean score over all frames
         self.assertAlmostEqual(results[0]['Cambi_feature_cambi_score'],
-                               0.8740225, places=4)
+                               0.79470425, places=4)
```

Comment:
Are these numerical change caused by SIMD??

Reply:
No, SIMD keeps the scores the same. But this PR also changes the window size from 63 to 65 which changes the scores. This is because 63 = 16*3+15 so it’s a bad fit for SIMD which operates in chunks of 16 items.

Reply:
In that case, it is best to do incremental changes: first PR changes the window size with numerical values; second PR changes to SIMD.

Also: if numerical changes, we need to update pycambi in vmaf_oss to make numerical values consistent.